api.md

```
@@ -0,0 +1,589 @@
+
+# API Docs
+
+### insert()
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)